nomad/constants.py

```diff
@@ -1,6 +1,12 @@
 # -*- coding: utf-8 -*-
 
 
+# PROJECTS
+# --
+
+METADATA_BASE_DIR = '.nomad'
```

Comment:
I really liked the idea of leveraging LXD's metadata system to avoid polluting project folders. I'm trying to see, in this changeset, what exactly justifies the creation of this folder, but I can't see it. Can you help me?